lib/livebook/application.ex

```diff
@@ -315,18 +315,24 @@ defmodule Livebook.Application do
   end
 
   defp create_online_hub(teams_key, name, org_id, org_key_id, agent_key) do
-    Livebook.Hubs.save_hub(%Livebook.Hubs.Team{
-      id: "team-#{name}",
-      hub_name: name,
-      hub_emoji: "💡",
-      user_id: nil,
-      org_id: org_id,
-      org_key_id: org_key_id,
-      session_token: agent_key,
-      teams_key: teams_key,
-      org_public_key: nil,
-      offline: nil
-    })
+    id = "team-#{name}"
+
+    hub =
+      Livebook.Hubs.save_hub(%Livebook.Hubs.Team{
+        id: id,
+        hub_name: name,
+        hub_emoji: "💡",
+        user_id: nil,
+        org_id: org_id,
+        org_key_id: org_key_id,
+        session_token: agent_key,
+        teams_key: teams_key,
+        org_public_key: nil,
+        offline: nil
+      })
+
+    Livebook.Hubs.set_default_hub(id)
```

Comment:
Instead of changing the default hub, let's have a separate config `Config.teams_auth_hub_id`. We can set it for both offline and online, since we can check if it's online in the caller side?

Reply:
I agree